src/cloupe/cloupe.py

```diff
@@ -0,0 +1,266 @@
+import os
+import csv
+import json
+import logging
+import struct
+import zlib
+
+from fontTools.misc.psOperators import ps_procmark
```

Comment:
what's this?

Reply:
I m also cing it for the first time, I don't know where that came in from. But I have removed it now.